lta/rest_server.py

```diff
@@ -0,0 +1,92 @@
+"""
+Long Term Archive REST API server.
+
+Run with `python -m lta.rest_server`.
+"""
+
+import asyncio
+import logging
+
+from tornado.ioloop import IOLoop
+
+from rest_tools.client import json_decode
+from rest_tools.server import RestServer, RestHandler, RestHandlerSetup, authenticated, catch_error
+
```

Comment:
```suggestion
import tornado
from functools import wraps

```

Reply:
You need this `import tornado` for Line 56

Reply:
ah, I see.  actually, I need `import tornado.web`